src/main/java/org/fundacionjala/coding/andy/DnaStrand.java

```diff
@@ -0,0 +1,19 @@
+package org.fundacionjala.coding.andy;
+
+/**
+ * @author lazaro on 10/28/2019.
+ * coding.
+ */
+public final class DnaStrand {
+    private DnaStrand() { }
+
+    public static String makeComplement(final String dna) {
```

Comment:
This may be re-written like this
```java
    public static String makeComplement(final String dna) {
        return dna.replace("A", "X")
                .replace("T", "A")
                .replace("X", "T")
                .replace("C", "X")
                .replace("G", "C")
                .replace("X", "G");
    }
```

Reply:
changed

Reply:
done